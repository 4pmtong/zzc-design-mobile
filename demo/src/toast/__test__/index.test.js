import React from 'react';
import { shallow, mount, render } from 'enzyme';
import { renderToJson } from 'enzyme-to-json';
import renderer from 'react-test-renderer';

import Demo from '../src/App.jsx';

test( 'toast', () => { 
  const wrapper = render( <Demo /> );
  expect(renderToJson(wrapper)).toMatchSnapshot();
} );