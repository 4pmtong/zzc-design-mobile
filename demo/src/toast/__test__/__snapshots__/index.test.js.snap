// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toast 1`] = `
<div
  class="zzc-demo"
>
  <div
    class="zzc-demo-header"
  >
    <h1
      class="zzc-demo-title"
    >
      Toast 轻提示
    </h1>
  </div>
  <div
    class="zzc-demo-body"
  >
    <div
      class="btn"
      id="noMarkInfo"
    >
      带有mark的info
    </div>
    <div
      class="btn"
    >
      不带有mark的info
    </div>
    <div
      class="btn"
    >
      success
    </div>
    <div
      class="btn"
    >
      error
    </div>
    <div
      class="btn"
    >
      waring
    </div>
    <div
      class="btn"
    >
      loading
    </div>
  </div>
</div>
`;
